src/SQLite.cs

```diff
@@ -2622,7 +3000,11 @@ public TableMapping (Type type, CreateFlags createFlags = CreateFlags.None)
 			_insertOrReplaceColumns = Columns.ToArray ();
 		}
 
-		private IReadOnlyCollection<MemberInfo> GetPublicMembers(Type type)
+		private IReadOnlyCollection<MemberInfo> GetPublicMembers(
+#if NET8_0_OR_GREATER
+			[DynamicallyAccessedMembers (DynamicallyAccessedMemberTypes.All)]
+#endif
+			Type type)
```

Comment:
We can't do this, because we're not looking for public properties, but only properties with public accessors. The properties themselves could have any accessibility, and we do need `All` to preserve privates on base types 🥲

Reply:
> we're not looking for public properties, but only properties with public accessors

But is it even possible to have a property which has an accessor that is less restrictive? My understanding is that since we're looking for a property with public getter and setter, it must be a public property ("The accessibility level on the accessor must be more restrictive than the accessibility level on the property or indexer itself." - https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/restricting-accessor-accessibility).

On the other hand, my suggestion goes against your idea of not making changes to the code, just make it work with NativeAOT. There will always be room for optimizations in follow-up PRs.